docs/common-toolchains.md

```diff
@@ -1,293 +1,212 @@
+| `2015b` | Jul '15 | 2.25       | 4.9.3 | 1.8.8      | *(none)*    | 0.2.14     | 3.5.0                 | 2.0.2       | 3.3.4       |
+| `2016a` | Jan '16 | 2.25       | 4.9.3 | 1.10.2     | *(none)*    | 0.2.15     | 3.6.0                 | 2.0.2       | 3.3.4       |
+| `2016b` | Jul '16 | 2.26       | 5.4.0 | 1.10.3     | *(none)*    | 0.2.18     | 3.6.1                 | 2.0.2       | 3.3.4       |
+| `2017a` | Jan '17 | 2.27       | 6.3.0 | 2.0.2      | *(none)*    | 0.2.19     | 3.7.0                 | 2.0.2       | 3.3.6(-pl2) |
+| `2017b` | Jul '17 | 2.28       | 6.4.0 | 2.1.1      | *(none)*    | 0.2.20\*   | (incl. with OpenBLAS) | 2.0.2       | 3.3.6(-pl2) |
+| `2018a` | Jan '18 | 2.28       | 6.4.0 | 2.1.2      | *(none)*    | 0.2.20\*   | (incl. with OpenBLAS) | 2.0.2       | 3.3.7       |
+| `2018b` | Jul '18 | 2.30       | 7.3.0 | 3.1.1      | *(none)*    | 0.3.1      | (incl. with OpenBLAS) | 2.0.2       | 3.3.8       |
+
+### Component versions in `intel` toolchain (deprecated versions)
+
+| `intel` | *date*  | *binutils* | *GCC* | *Intel compilers* | *Intel MPI* | *Intel MKL* |
+|---------|---------|------------|-------|-------------------|-------------|-------------|
+| `2014b` | Jul '14 | '*(none)*  | 4.8.3 | 2013.5.192        | 4.1.3.049   | 11.1.2.144  |
+| `2015a` | Jan '15 | '*(none)*  | 4.9.2 | 2015.1.133        | 5.0.2.044   | 11.2.1.133  |
+| `2015b` | Jul '15 | 2.25       | 4.9.3 | 2015.3.187        | 5.0.3.048   | 11.2.3.187  |
+| `2016a` | Jan '16 | 2.26       | 4.9.3 | 2016.1.150        | 5.1.2.150   | 11.3.1.150  |
+| `2016b` | Jul '16 | 2.26       | 5.4.0 | 2016.3.210        | 5.1.3.181   | 11.3.3.210  |
+| `2017a` | Jan '17 | 2.27       | 6.3.0 | 2017.1.132        | 2017.1.132  | 2017.1.132  |
+| `2017b` | Jul '17 | 2.28       | 6.4.0 | 2017.4.196        | 2017.3.196  | 2017.3.196  |
+| `2018a` | Jan '18 | 2.28       | 6.4.0 | 2018.1.163        | 2018.1.163  | 2018.1.163  |
+| `2018b` | Jul '18 | 2.30       | 7.3.0 | 2018.3.222        | 2018.3.222  | 2018.3.222  |
+
+## Customizing common toolchains
+
+Sometimes the need arises to customize one or more components of a
+common toolchain w.r.t. site-specific aspects. One common example is
+using additional configuration options for Open MPI.
+
+To customize a toolchain component, you should copy the corresponding
+easyconfig file, modify according to your needs, and make sure it is
+included in a location in the robot search path that precedes the
+location of the easyconfig files that are included with EasyBuild (see
+also [Searching for easyconfigs: the robot search path](using-easybuild.md#searching-for-easyconfigs-the-robot-search-path)), before building and
+installation the toolchain.
+
+!!! info "More information about toolchains"
+
+    Please see the [List of known toolchains](version-specific/eb-list-toolchains.md#list-of-known-toolchains) for how to obtain a listing of
```

Comment:
I've added links in this section that point at content within the auto-generated `version-specific` directory. I've taken the placeholders from the current docs' pages but is this the correct thing to do? Obviously these placeholders don't yet exist as we're deliberately not editing any content in this directory.

Reply:
Resolved by 9ef3db1